x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/utils.test.ts

```diff
@@ -16,12 +16,18 @@ import {
   transformAlertsToRules,
   transformOrImportError,
   getDuplicates,
+  getTupleDuplicateErrorsAndUniqueRules,
 } from './utils';
 import { getResult } from '../__mocks__/request_responses';
 import { INTERNAL_IDENTIFIER } from '../../../../../common/constants';
-import { OutputRuleAlertRest } from '../../types';
+import { OutputRuleAlertRest, ImportRuleAlertRest } from '../../types';
 import { BulkError, ImportSuccessError } from '../utils';
 import { sampleRule } from '../../signals/__mocks__/es_results';
+import { getSimpleRule } from '../../../../../../../../test/detection_engine_api_integration/security_and_spaces/tests/utils';
```

Comment:
As advice as previously about mixing test/utils. We should just move `getSimpleRule` here and instead change the e2e tests to import from here so we don't have cyclic imports and to make it more simple.

Reply:
👍 updated!